daemon/response.go

```diff
@@ -124,6 +124,8 @@ const (
 	errorKindTwoFactorFailed   = errorKind("two-factor-failed")
 	errorKindLoginRequired     = errorKind("login-required")
 	errorKindInvalidAuthData   = errorKind("invalid-auth-data")
+	errorKindTOSNotAccepted    = errorKind("tos-not-accepted")
```

Comment:
Can we please call this errorKindTermsNotAccepted and "terms-not-accepted"?

Minor difference, but we may want to use that same kind elsewhere.